Dockerfile

```diff
@@ -1,27 +1,21 @@
 FROM golang:1.20.6-alpine AS builder
-ARG VERSION=unknown
-ARG COMMIT=unknown
 
+RUN echo "badrobot:x:25000:25000:badrobot:/home/badrobot:/sbin/nologin" > /passwd && \
+    echo "badrobot:x:25000:" > /group
 WORKDIR /badrobot
-
+RUN apk add --no-cache ca-certificates
 COPY . .
-
 RUN CGO_ENABLED=0 GOOS=linux go build \
     -ldflags="-s -w -X github.com/controlplaneio/badrobot/cmd.version=$VERSION -X github.com/controlplaneio/badrobot/cmd.commit=$COMMIT" \
     -o badrobot .
 
 # ===
 
-FROM alpine:3.18.2
-
-RUN addgroup -S badrobot \
-    && adduser -S -g badrobot badrobot \
-    && apk --no-cache add ca-certificates
-
+FROM scratch
 WORKDIR /home/badrobot
-
+COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+COPY --from=builder /passwd /group /etc/
```

Comment:
Do we actually need to copy this stuff across, I think we can just set the user to 1000 or whatever and we are good to go.

Reply:
No passwd and group can work but you'll have an unregistered user

```
λ docker run -u 123456 -it debian:bookworm-slim
I have no name!@ee4c720e0524:/$
```

And yes you'd have to do `USER 25000` rather than `USER badrobot` or launching the container would fail

IMO the extra 8K to the container (that then gets compressed) is that big a negative to skip this

```
λ  echo "badrobot:x:25000:25000:badrobot:/home/badrobot:/sbin/nologin" > /tmp/passwd && \
    echo "badrobot:x:25000:" > /tmp/group
λ du -h /tmp/{passwd,group}
4.0K	/tmp/passwd
4.0K	/tmp/group
```

Reply:
alright, lets leave it as is